ckanext/who_afro/assets/css/who-afro.css

```diff
@@ -876,11 +882,7 @@ p.small {
     margin-bottom:  24px;
 }
 
-.insight-square:first-child {
-    margin-right: 24px;
-    width: calc(33.3% - 24px) ;
-}
-.insight-square:nth-child(2) {
+.insight-square {
```

Comment:
This doesn't break anything on the home page?  Looks odd to me that you can make this change and not break something, but maybe I am missing something.